### DISCUSSION TITLE
Shared view with Write access [Apr 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5595] [Discord Threads]
> 🙋‍♂️ **semighoti**  
## Hi guys, I have a question, is there a way to lock the configuration but allow editors to edit tables?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1090036486663057499/Screenshot_from_2023-03-27_15-01-26.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Shared view is currently read only. It would solve your purpose if they are write enabled. There was a GitHub request too on this.    
####    
#### OR this can be sorted out via Row Level Access Control. Not supported as of now.    
####    
#### I couldn't think of any existing mechanism to get through your requirement.  
#  
> 🙋‍♂️ **semighoti**  
#### I agree that shared view should be read only. Because we share tables with guests, so let it be   
#### I have heard about the RLA , and not sure if my question regarded it or not   
#### I meant that it would be nice if we can Lock the view configuration in Collaboration (Lock View) but not locking cells so editors can edit it.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1090036486868582581)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
